src/tests/ftest/dfuse/il_whitelist.py

```diff
@@ -0,0 +1,86 @@
+"""
+  (C) Copyright 2021-2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import os
+
+from apricot import TestWithServers
+from command_utils_base import EnvironmentVariables
+from dfuse_utils import get_dfuse, start_dfuse
+from host_utils import get_local_host
+from run_utils import run_remote
+
+
+class ILWhiteList(TestWithServers):
+    """Run whitelist_test and check the number of daos_init().
+
+    :avocado: recursive
+    """
+
+    def run_test(self, il_lib='libpil4dfs.so', interception=False):
+        """Jira ID: DAOS-15583.
+
+        Test Description:
+            Mount a DFuse mount point
+            Run whitelist_test.
+            Check the number of daos_init() with interception library
+        """
+        env = EnvironmentVariables()
+        if il_lib is not None:
+            env["LD_PRELOAD"] = os.path.join(self.prefix, "lib64", il_lib)
+
+        exe_path = os.path.join(self.prefix, "lib", 'daos/TESTING/tests/whitelist_test')
```

Comment:
I think either
```suggestion
        exe_path = os.path.join(self.prefix, 'lib/daos/TESTING/tests/whitelist_test')
```
or
```suggestion
        exe_path = os.path.join(self.prefix, "lib", 'daos', 'TESTING', 'tests', 'whitelist_test')
```

Reply:
Thank you very much! I will revise it with the first suggested form then.

Reply:
Fixed as you suggested. Thank you!